test/mpi/runtests

```diff
@@ -42,7 +42,7 @@
 # Import the mkpath command
 use strict;
 use File::Path;
-use File::Copy qw(move);
+use File::Copy qw(mv);
```

Comment:
Isn't `mv` an alias for `move`? -- https://perldoc.perl.org/File::Copy. Is there a reason to use `mv` instead of `move`? If there is, please add a comment.

Reply:
The documentation states:
> You may use the mv alias for this function in the same way that you may use the cp alias for copy.

and:
> The syntax is exactly the same. The behavior is nearly the same as well: as of version 2.15, `cp` will preserve the source file's permission bits like the shell utility `cp(1)` would do with default options, while copy uses the default permissions for the target file (which may depend on the process' umask, file ownership, inherited ACLs, etc.).

When using `move` instead, I observed that the executable bit was not set for the test binaries. I added an according comment.